README.md

```diff
@@ -0,0 +1,41 @@
+# Cincinnati
+
+Cincinnati is a service to transform update information as modeled in a Docker registry (such as Quay.io) into a graph representation that can be consumed by update clients. It is the spiritual successor to the Omaha protocol used to udate Google Chrome and Google Chrome OS based operating systems.
+
+## Common Usage
+
+```sh
+# docker run -d -p 8080:8080 <image> --address 0.0.0.0 --registry https://quay.io --repository <namespace/reponame>
+# docker run -d -p 8081:8081 --entrypoint policy-engine <image> --address 0.0.0.0 --upstream http://localhost:8080/v1/graph
+# curl -v -H 'Accept:application/json' http://localhost:8080/graph
+```
+
+## Components
+
+### Graph Bulder
+
+The Graph Builder is responsible for transforming the data in a Docker registry repository into a directed acyclic graph (DAG) describing update payloads as nodes, and their relationship as edges. 
+
+There is one repository that serves as a command and control repository for each update payload. Each tag in the command and control repository is pulled individually, and each layer is inspected for a `cincinnati.json` file until one is found. The contents of the `cincinnati.json` file describe the update payload for that label, and provide metadata about the update payload itself.
+
+### cincinnati.json
```

Comment:
I would leave this off. This seems too detailed for a README.

Reply:
It has to be documented somewhere... I had to reverse engineer it from the serde definition.

Reply:
There are internal documents that detail its structure. I'm in the process of adding these directly to the repo, but it's not as simple as copying and pasting.